### PR TITLE
fix: decimal issue that was causing bridged USDC balance error

### PR DESCRIPTION
**Pull request by bowd**
Closes: #31 

## Overview

This PR tackles two separate but somewhat related issues:
1. The website doesn't correctly count USDC in the Mento Reserve Multisig
2. The website doesn't account for Celo or stCelo held directly by the Reserve Multisig

(1) The USDC issue is actually a decimal one. The balance of the ERC20 token was flowing through a `formatNumber` function that assumed 18 decimals for all values, therefore it was counting the USDC but the value was scaled wrongly so didn't seam to be in the total. Extending the function to support optional number of decimals fixes the issue. 

(2) I have manually added the reserve multisig content to the `custody` calculation logic. To be clear this won't account for Celo/stCelo included in Uniswap pools, just held directly by the address.

## Other changes

- Renamed some constants for clarity
- Included USDC_AXELAR in accounting

### CODE REVIEW

File: src/helpers/StakedCeloProvider.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { JsonRpcProvider } from "@ethersproject/providers"
+import BigNumber from "bignumber.js"
+import { ethers } from "ethers"
+
+const STAKED_CELO_MANAGER_ADDRESS = "0x0239b96D10a434a56CC9E09383077A0490cF9398"
+const STAKED_CELO_ERC20_ADDRESS = "0xC668583dcbDc9ae6FA3CE46462758188adfdfC24"
+
+const STAKED_CELO_ERC20_ABI = [
```
Comment (philbow61) [inline src/helpers/StakedCeloProvider.ts:8]: there is already a ERC20_ABI in src/constants/abis.ts
to keep the code consistent i would this same for the addresses in src/contract-addresses.ts

**Review by philbow61 — CHANGES_REQUESTED** [review]
LGTM 🚢 suggested one little refactoring

**Review by ninocomputer — APPROVED** [review]
looks good to me!

File: src/helpers/StakedCeloProvider.ts (hunk 1 above)
Comment (bowd) [reply src/helpers/StakedCeloProvider.ts]: Good point! Made the changes!

**Review by philbow61 — APPROVED** [review]

**Pull request merged**